### PR TITLE
Bratseth/mail rag

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+layout: post
+title: "Yahoo Mail turns to Vespa to do RAG at scale"
+author: bratseth
+date: '2023-11-01'
+image: assets/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale/evgeni-tcherkasski.jpg
+image_credit: 'Photo by <a href="https://unsplash.com/@evgenit?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Evgeni Tcherkasski</a> on <a href="https://unsplash.com/photos/red-and-black-bridge-over-water-XBJtNRyZUx4?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+'
+skipimage: true
+tags: [rag, vectors, streaming]
+excerpt: Vespa is becoming its own company!
```
Comment (kkraune) [inline _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md:11]: wrong excerpt? (that goes for yesterday's post, too I just noticed)

File: _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+--- 
+layout: post
+title: "Yahoo Mail turns to Vespa to do RAG at scale"
+author: bratseth
+date: '2023-11-01'
```
Comment (kkraune) [inline _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md:5]:
```suggestion
date: '2023-11-02'
```

File: _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+tags: [rag, vectors, streaming]
+excerpt: Vespa is becoming its own company!
+---
+
+Yahoo Mail is one of the largest mail providers in the world. Now they’re also taking a shot at being the most 
+innovative with their recent release of AI-driven features which lets you 
+[ask questions of your mailbox](https://www.fastcompany.com/90945096/yahoo-unveils-an-ai-email-assistant-and-it-works-with-gmail)
+and tell it to do things for you.
+
+![Yahoo Mail](/assets/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale/video2.mp4 "image_tooltip")
```
Comment (kkraune) [inline _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md:19]: Had to do this to load and autoplay video:
```suggestion
<video autoplay muted preload="auto">
  <source src="/assets/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale/video2.mp4" type="video/mp4">
  Your browser does not support the video tag.
</video>
```

**Review by kkraune — APPROVED** [review]
Great post! LGTM with the suggestions

File: _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md (hunk 1 above)
Comment (bratseth) [reply _posts/2023-11-02-yahoo-mail-turns-to-vespa-to-do-rag-at-scale.md:11]: "Excerpt" what's that? I'll fix :-)

**Comment by bratseth** [comment]
Thanks!

**Pull request merged**